### PR TITLE
[DEX] Misc UI Fixes

### PR DESCRIPTION
**Pull request by crishellco**
## Ticket
* n/a

## Changes
* UI clean up

## Screenshots
n/a

## Code Coverage
n/a

### CODE REVIEW

File: src/components/dex/OrderBook.vue [hunk 1]
```diff
@@ -80,14 +81,15 @@ export default {
       display: flex;
       flex-direction: column;
       flex: 1;
-      margin: $space;
+      margin: 0   $space;
```
Comment (schaeferCode) [inline src/components/dex/OrderBook.vue]: Is there an extra space between `0` and `$space`?

File: src/store/mutations.js [hunk 2]
```diff
@@ -443,6 +444,10 @@ function SOCKET_ONCLOSE(state) {
   }
 }
 
+function SOCKET_ONERROR() {
+  // Socket error...
```
Comment (schaeferCode) [inline src/store/mutations.js:448]: Do we not have the code that should go here?

File: src/components/modals/DepositWithdrawModal.vue [hunk 3]
```diff
@@ -1,43 +1,42 @@
 <template>
-  <section class="aph-deposit-withdraw-modal">
-    <div class="content">
-      <div class="icons">
-        <aph-icon name="hex"></aph-icon>
-        <aph-icon name="trade"></aph-icon>
+  <modal-wrapper id="aph-deposit-withdraw-modal" icon="dex">
```
Comment (schaeferCode) [inline src/components/modals/DepositWithdrawModal.vue:2]: Nice! Smart call on the modal wrapper.

**Review by schaeferCode — APPROVED** [review]

File: src/store/mutations.js (hunk 2 above)
Comment (crishellco) [reply src/store/mutations.js:448]: Correct. This is just to stop console errors.

**Pull request merged**